src/interpreter/utils.ts

```diff
@@ -417,7 +417,7 @@ export function findTypescriptProperty(name: string, node: ts.Node): boolean {
 }
 
 export function getTypescriptIdentifierName(node: ts.Node): string {
-  if (ts.isIdentifier(node)) {
+  if (ts.isIdentifier(node) || ts.isStringLiteral(node)) {
```

Comment:
question: Why is this change needed?

Reply:
Added it because of another change (was passing `argumentExpression` which could be string literal), but looking at references of this function it wasn't used at all, even though it was doing the same thing as `getVariableName` function, so I moved this logic there.